docs/developer/game-loop.md

```diff
@@ -26,21 +26,21 @@ Each frame, the following steps will occur (in order):
 
 ### Physics Step #physics
 
-The physics step is the step in which sprites move around the screen based off their velocity and acceleration, and any ``||scene:collisions||`` and ``||sprites:overlaps||`` are calculated. The movements in this step are interpolated into small steps for fast moving ``||sprites:sprites||``, to avoid ``||sprites:sprites||`` moving through walls or missing ``||sprites:sprite overlap events||``.
+The physics step is the step in which sprites move around the screen based on their velocity and acceleration, and any ``||scene:collisions||`` and ``||sprites:overlaps||`` are calculated. The movements in this step are interpolated into small increments for fast moving ``||sprites:sprites||`` inorder to avoid ``||sprites:sprites||`` moving through walls or missing ``||sprites:sprite overlap events||``.
 
-1. Each sprite in the scene has it's velocity updated by it's acceleration and the maximum speed set for the current physics engine. These velocities are then split up into small steps that will be taken once at a time, and stored for each sprite.
+1. Each sprite in the scene has it's velocity updated by it's acceleration and the maximum set speed for the current physics engine. These velocities are then split up into small steps that will be taken once at a time, and stored for each sprite.
 2. Obstacles are cleared from each sprite that is moving.
 3. Each sprite is moved one step:
-    1. The stored velocities are compared to the sprite's current velocities, in case some event has caused the sprite to turn around. If the sprite has turned around, adjust the movement so that the sprite doesn't move in the wrong direction.
+    1. The stored velocities are compared to the sprite's current velocities, in case some event has caused the sprite to turn around. If the sprite has turned around, the movement is adjusted so that the sprite doesn't move in the wrong direction.
     2. The sprite is moved a few pixels according to it's speed.
     3. If the sprite is not a ghost and a tile map exists, the sprite is checked for collisions with the tile map. If it has hit something, the sprite may be stopped and an collision event with that type of wall is run.
-    4. If the sprite is not finished moving, it is stored to move more steps.
+    4. If the sprite has not finished moving, it is stored to move again in future steps.
 4. After all the sprites have moved, sprite overlaps are checked, and any overlap events between sprites that are not currently running are set to run [independent from the game loop](#fibers).
-5. If there are sprites that are not finished moving, return to step 3 and continue moving the sprites. Otherwise, the physics movement is complete for this frame.
+5. If there are sprites that still need to move, step 3 is repeated to continue moving the sprites. Otherwise, the physics movement is complete for this frame.
```

Comment:
Hm, does this have the same connotation? It's pretty obvious what it's meant to mean I think, but a strict (antagonistic?) reading could potentially be to just repeat step 3 and not check overlaps in step 4 again.

Reply:
It's OK but just wanted to clarify that sprites with a move delta remaining after that cycle are moved again by going back to step 3 and moving them until their movement is finished. Say that in ten words or less.

Reply:
Yeah, I can't get the wording right for this in my head. `Repeat steps 3 and 4 until all sprites have completed their movement for this frame.` maybe? Or just leave it, its understandable i think..

Reply:
For now, there are better things to agonize over.

Reply:
Make a step/flow diagram. Screen cap of visio or something. Like you have time for that.

Reply:
Or, do a step frame animation with functional blocks and state positions. Similar in fashion to what I did here: https://makecode.adafruit.com/learnsystem/pins-tutorial/devices/capacitors

Reply:
Ooh, thats a good idea, lll try to dig out my graphics tablet and draw something later tonight

Reply:
I can help with the animation stuff if you want.

Reply:
@ganicke I think for now I'm just gonna merge this one as is for now, and we can add visuals in a separate PR ~